charts/ziti-router/values.yaml

```diff
@@ -94,6 +94,7 @@ tunnel:
   # -- built-in nameserver configuration, e.g. udp://127.1.2.3:53
   resolver: none
   # lanIf: eth0   # interface device name for tproxy?
+  # dnsSvcIpRange: 100.72.0.0/13 # subnet that is used for the internal service name resolution
```

Comment:
Instead of commenting the value, let it be null by default (not assigned). 

This way, you can document the value and it will automatically appear in the README's table of inputs.

Add a comment above the value like `# -- this comment documents the value's purpose and expected type`.

Will you also document `lanIf`?

Reply:
lanIf is not really used in k8s